contents/data/sql/SQL.md

```diff
@@ -0,0 +1,145 @@
+<frontmatter>
+  title: Introduction to SQL
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to SQL
+
+**Author(s): [Amrut Prabhu](https://github.com/amrut-prabhu)**
+
+Reviewers: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Rahul Rajesh](https://github.com/rrtheonlyone)
+
+## What is SQL?
+
+Databases are integral to any commercial software application, whether it is a web app, desktop app or otherwise.
```

Comment:
This statement is not accurate. 1. Being commercial has no relevant 2. There can be apps that doesn't use a database at all. You don't want to open with such a questionable statement.